packages/core/src/config/parse.ts

```diff
@@ -360,16 +425,40 @@ export const parseInplaceBytes: VariableHandler<ParsedConfigVariable> = (
  * @param props standard VariableHandler props. See ./iterator.ts for more information.
  * @returns
  */
-export const parseInplaceUint: VariableHandler<ParsedConfigVariable> = (
-  props: VariableHandlerProps<ParsedConfigVariable>
-) => {
-  const { variable, variableType } = props
+export const parseInplaceUint: VariableHandler<UserConfigVariable, string> = (
+  props: VariableHandlerProps<UserConfigVariable, string>
+): string => {
+  const { variable, variableType, storageObj } = props
+
+  if (
+    typeof variable !== 'number' &&
+    typeof variable !== 'string' &&
+    !(
+      typeof variable === 'object' &&
+      'type' in variable &&
+      variable.type === 'BigNumber'
+    )
```

Comment:
Can use [`ethers.isBigNumber`](https://docs.ethers.org/v5/api/utils/bignumber/#BigNumber--BigNumber--methods--inspection)

Reply:
Believe it or not, this actually doesn't work. I think it's because of how we handle importing the config object.